types/handler_test.go

```diff
@@ -30,10 +30,18 @@ func (s *handlerTestSuite) TestChainAnteDecorators() {
 	mockCtrl := gomock.NewController(s.T())
 	mockAnteDecorator1 := mocks.NewMockAnteDecorator(mockCtrl)
 	mockAnteDecorator1.EXPECT().AnteHandle(gomock.Eq(ctx), gomock.Eq(tx), true, gomock.Any()).Times(1)
-	sdk.ChainAnteDecorators(mockAnteDecorator1)(ctx, tx, true) //nolint:errcheck
+	_, err := sdk.ChainAnteDecorators(mockAnteDecorator1)(ctx, tx, true)
+	s.Require().NoError(err)
 
 	mockAnteDecorator2 := mocks.NewMockAnteDecorator(mockCtrl)
-	mockAnteDecorator1.EXPECT().AnteHandle(gomock.Eq(ctx), gomock.Eq(tx), true, mockAnteDecorator2).Times(1)
-	mockAnteDecorator2.EXPECT().AnteHandle(gomock.Eq(ctx), gomock.Eq(tx), true, nil).Times(1)
-	sdk.ChainAnteDecorators(mockAnteDecorator1, mockAnteDecorator2)
```

Comment:
There is something wrong here: this tests can't pass because in the last case the ante chain is not even called (`sdk.ChainAnteDecorators` return a decorator, but we still need to call it). 

I added a call but we can't correctly check the `mockAnteDecorator2` as a expected argument - see the note in the code.